mappings/empty-case-list.json

```diff
@@ -15,6 +15,7 @@
     },
     "status": 200,
     "jsonBody": {
+      "lastUpdated": "2019-12-03T08:30:00",
```

Comment:
Actually, might be better to change this to use the current date:

"lastUpdated": "{{now format='yyyy-MM-dd'}}T08:30:00",